yt/visualization/fixed_resolution_filters.py

```diff
@@ -53,30 +58,24 @@ class FixedResolutionBufferGaussBeamFilter(FixedResolutionBufferFilter):
     """
     This filter convolves
     :class:`yt.visualization.fixed_resolution.FixedResolutionBuffer` with
-    2d gaussian that is 'nbeam' pixels wide and has standard deviation
-    'sigma'.
+    2d gaussian that has standard deviation 'sigma'. Extra arguments
+    are directly passed to `scipy.ndimage.gaussian_filter`.
     """
     _filter_name = 'gauss_beam'
 
-    def __init__(self, nbeam=30, sigma=2.0):
-        self.nbeam = nbeam
+    def __init__(self, sigma=2.0, nbeam=None, **kwa):
+        if nbeam is not None:
```

Comment:
Since you're deprecating `nbeam`, maybe it should be looked for in the kwargs instead.
Furthermore, it would feel more consistent to store a `self.truncate` attribute instead of passing it among kwargs.